backend.py

```diff
@@ -18,14 +19,10 @@ def hello():
 
 @app.route('/log')
 def log():
-    cursor.execute("SELECT * FROM TestTable WHERE id = 1;")
+    cursor.execute("SELECT Lang_Desc FROM Lang_Ref;")
     conn.commit()
-    data = cursor.fetchall()
-    (key, word, sentance) = data[0]
-    now = datetime.now()
-    file1 = open("log.txt","w+")
-    file1.write(f"This was pulled from the database {sentance} and it was done at {now}")
-    return render_template('index.html')
+    language_list = logic.tuple2list(cursor.fetchall()) # Uses function from logic file to convert tuple to list 
+    return render_template('index.html', language_list = language_list)
```

Comment:
Can you put this inside hello() function since we need it on landing page? The logic looks good.